common/transport/snmp/src/main/java/org/thingsboard/server/transport/snmp/service/SnmpTransportService.java

```diff
@@ -269,12 +267,8 @@ public void onError(Throwable e) {
         }
     }
 
-    @TbSnmpTransportComponent
-    @Configuration
-    static class Config {
-        @Bean
-        public TransportName transportName() {
-            return new TransportName("snmp");
-        }
+    @Override
+    public String getName() {
+        return "snmp";
```

Comment:
Let's use capital letters for the transport names. In most cases, this is an abbreviation.